clients/client-java/src/test/java/com/datastrato/gravitino/client/TestOAuth2TokenProvider.java

```diff
@@ -31,6 +31,7 @@
 import org.mockserver.matchers.Times;
 import org.mockserver.model.HttpResponse;
 
+@SuppressWarnings("JavaUtilDate")
```

Comment:
`Date` comes from `io.jsonwebtoken.JwtBuilder#setExpiration`. I can't figure out a way to fix it. So I just ignore it.

```
/Users/dian/Project/gravitino/clients/client-java/src/test/java/com/datastrato/gravitino/client/TestOAuth2TokenProvider.java:126: 警告: [JavaUtilDate] Date has a badPI that leads to bugs; prefer java.time.Instant or LocalDate.
            .setExpiration(new Date(System.currentTimeMillis() + 10000))
                           ^
    (see https://errorprone.info/bugpattern/JavaUtilDate)

```